packages/block-library/src/post-comment-date/index.php

```diff
@@ -19,14 +19,21 @@ function render_block_core_post_comment_date( $attributes, $content, $block ) {
 	}
 
 	$wrapper_attributes = get_block_wrapper_attributes();
+	$formatted_date     = get_comment_date(
+		isset( $attributes['format'] ) ? $attributes['format'] : '',
+		$block->context['commentId']
+	);
+	$link               = get_comment_link( $block->context['commentId'] );
+
+	if ( isset( $attributes['isLink'] ) && $attributes['isLink'] ) {
```

Comment:
`empty` is a shorter versio of the same check:
```suggestion
	if ( ! empty( $attributes['isLink'] ) ) {
```